### PR TITLE
docs: make the document easier to read about upstream host.

### PR DESCRIPTION
**Pull request by moonming**
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Comment by moonming** [comment]
@juzhiyuan please update the dashboard repo.

**Comment by juzhiyuan** [comment]
ok

**Review by nic-chen — APPROVED** [review]

File: docs/zh/latest/admin-api.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ title: Admin API
 
 ## 目录
 
-- [目录](#目录)
+- [目录](#%e7%9b%ae%e5%bd%95)
```
Comment (spacewander) [inline docs/zh/latest/admin-api.md:26]: Please keep the original name

File: docs/en/latest/admin-api.md [hunk 2]
```diff
@@ -547,11 +547,11 @@ In addition to the basic complex equalization algorithm selection, APISIX's Upst
 `hash_on` can be set to different types:
 
 1. when it is `vars`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html), without the prefix '$'.
-1. when it is `header`, the `key` is required. It is equal to "http_`key`".
-1. when it is `cookie`, the `key` is required. It is equal to "cookie_`key`".
-1. when it is `consumer`, the `key` is optional. The key is the `consumer_name` set by authentication plugin.
-1. when it is `vars_combinations`, the `key` is required. The `key` can be any [Nginx builtin variables](http://nginx.org/en/docs/varindex.html) combinations, such as `$request_uri$remote_addr`.
-1. If there is no value for either `hash_on` or `key`, `remote_addr` will be used as key.
+2. when it is `header`, the `key` is required. It is equal to "http_`key`".
```
Comment (spacewander) [inline docs/en/latest/admin-api.md:550]: We don't need to change this. Markdown will handle the order automatically. There is no need to maintain the order by us.

File: docs/zh/latest/admin-api.md (hunk 1 above)
Comment (moonming) [reply docs/zh/latest/admin-api.md]: thanks

File: docs/en/latest/admin-api.md (hunk 2 above)
Comment (moonming) [reply docs/en/latest/admin-api.md]: yes, fixed

File: docs/en/latest/admin-api.md [hunk 3]
```diff
@@ -23,15 +23,15 @@ title: Admin API
...
-* [Plugin Config](#plugin-config)
-* [Plugin Metadata](#plugin-metadata)
-* [Plugin](#plugin)
+- [Route](#route)
+- [Service](#service)
+- [Consumer](#consumer)
+- [Upstream](#upstream)
+- [SSL](#ssl)
+- [Global Rule](#global-rule)
+- [Plugin config](#plugin-config)
```
Comment (juzhiyuan) [inline docs/en/latest/admin-api.md:32]: Why we need to use lowercase here?

**Review by spacewander — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]
Others LGTM

**Review by spacewander — APPROVED** [review]

**Pull request merged**